src/css/styles.scss

```diff
@@ -1,167 +1,140 @@
-/*
-// Load CSS Reset from NPM
-// CSS resets "normalize" how browser styles are applied.
-// Try commenting this out and saving. What changes in the browser?
-//@import "~normalize.css";//
-
-// An example of a partial.
-// Check out the _typography.scss file to edit the CSS inside!
-//@import "typography";//
-
-// You can still write styles in this file too.
-// Here's an example of how you could write media queries more easily with Sass.
-
-//$large-phone: 500px;//
-//$tablet-vertical: 768px;//
-//$desktop: 1200px;//
-
-// And some color variables.
-// Look how much easier it is to keep track of what these are than hex codes!
-
-//$red: #ab424a;//
-//$orange: #dd7c00;//
-//$green: #3b975f;//
-//$blue: #4261ab;//
-
-//.page-heading {//
-	//color: $red;//
-
-	// Sass lets you write your media queries inline if you want.
-	// This is why I'm always forgetting I have to put the class inside
-	// the media query in regular CSS. I never have to in Sass!
-	// I like combining these with variables to make it easier to read
-	// my media queries.
-
-	//@media screen and (min-width: $large-phone) {//
-		//color: $orange;//
-	//}//
-
-	//@media screen and (min-width: $tablet-vertical) {//
-		//color: $green;//
-	//}//
-
-	//@media screen and (min-width: $desktop) {//
-		//color: $blue;//
-	//}//
-//}//
-
-*/
+.nav_home::after {
+	content: '';
+	width: 100%;
+	height: 100%;
+	border-top: solid 50px #e9c4bc;
+	position: absolute;
+	left: 0;
+	top: 50px;
+	z-index: 1;
+}
 
-body {
-	background: white;
+.nav_home {
+	list-style-type: none;
 	margin: 0;
 	padding: 0;
+	overflow: hidden;
 }
 
-h1 {
-	color: #323536;
-	font-family: 'Inria Serif', 'Roboto', serif;
-	font-weight: 100;
-	font-size: 400%;
+li {
+	float: right;
 }
 
-p {
-	color: #323536;
-	font-family: 'Inria Serif', 'Roboto', serif;
-	font-size: 150%;
+li a {
+	display: block;
+	color: black;
```

Comment:
I would recommend using hex colors instead of color keywords in your final version - they provide more flexibility in the color value, and you'll know exactly how the color is applied across browsers. More information here: https://stackoverflow.com/questions/3230851/are-there-any-cons-to-using-color-names-in-place-of-color-codes-in-css